package.json

```diff
@@ -30,7 +30,9 @@
     "preventManualRelease": "node scripts/preventManualRelease.js",
     "buildHooksIndex": "node scripts/createIndexFile.js ./packages/hooks/src",
     "lerna:clean": "lerna run clean && lerna clean --y && npm run clean:packages && npm run clean",
-    "clean:packages": "rm -rf node_modules **/dist"
+    "clean:packages": "rm -rf node_modules **/dist",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Maybe we could change the port to not conflict with jobber-mobile's storybook so we can have both instances running?

Reply:
oh that's a good idea. `6009`? 😏